sql-plugin/src/main/java/com/nvidia/spark/rapids/catalyst/CudfUnsafeRow.java

```diff
@@ -14,9 +14,10 @@
  * limitations under the License.
  */
 
-package org.apache.spark.sql.catalyst;
+package com.nvidia.spark.rapids.catalyst;
```

Comment:
Nit: Why isn't this just under com.nvidia.spark.rapids?  The `com.nvidia.spark.rapids.catalyst` package doesn't exist before this PR, and arguably many things under the former would be candidates for the latter since they deal with Catalyst.

Reply:
Agreed.  I will move it to com.nvidia.spark.rapids